webcompat/form.py

```diff
@@ -216,7 +223,7 @@ def build_formdata(form_object):
 **Steps to Reproduce**
 {description}
 
-'''.format(**formdata)
+""".format(**formdata)
```

Comment:
@karlcow ? I had to look up format but still am not clear. is this converting the formdata dictionary into a string? (pardon me if I ever mix js and python terms b/c it has been a while since using python daily). my high level gist of the build_formdata() function is that this is where you are building the string that stores the bug form field values.

on a side note. is it considered unimportant whether a user would want to edit the issue field values in the future? I may have missed something but it seemed like you could only edit an existing issue by adding comments(?)